package.json

```diff
@@ -10,27 +10,17 @@
     "test:components": "npx cross-env NODE_ENV=test PORT=8080 ENV_ENVIRONMENT='development' ENV_SECRET='mwmBqNcTWiKxDHygMiKhwyffaKQCVoRQ' nyc --reporter=lcov --reporter=text ava `find components/ -name \"*.spec.js\"`",
     "test:lib": "npx cross-env NODE_ENV=test PORT=8080 ENV_ENVIRONMENT='development' ENV_SECRET='mwmBqNcTWiKxDHygMiKhwyffaKQCVoRQ' nyc --reporter=lcov --reporter=text ava `find lib/ -name \"*.spec.js\"`",
     "watch:test": "npm run test -- --watch",
-    "end-to-end-test": "run-p -r dev testcafe-tests",
-    "testcafe-tests": "wait-on -t 60000 -w 2000 -l http://localhost:3122 && node node_modules/testcafe/bin/testcafe.js --skip-js-errors firefox:headless systemtest/landing.test.js",
-    "lint": "standard",
-    "fix": "standard --fix",
-    "dev": "cross-env ENV_ENVIRONMENT='development' ENV_SECRET='mwmBqNcTWiKxDHygMiKhwyffaKQCVoRQ' nodemon -w lang -w server -w package.json server/server.js",
```

Comment:
Would recommend keeping, lint, fix, & dev.

Reply:
Oh ya I just moved them down I think?

Reply:
we can take out the testcafe-tests
I want to replace them with codecept tests